App/Screens/Screens.tsx

```diff
@@ -139,5 +139,10 @@ export function Screens(): React.ReactElement {
 
   const stack = renderScreen(api, error);
 
-  return <View style={theme.fullScreen}>{stack}</View>;
+  return (
+    <View style={theme.fullScreen}>
+      <StatusBar translucent={false} />
```

Comment:
Could you actually put this in `App.tsx`?

Reply:
Sure, I'm done with it yet